docs-content/index.html.md.erb

```diff
@@ -1,18 +1,23 @@
 ---
-title: Datadog Nozzle for PCF
+title: Datadog Cluster Monitoring for PCF
 owner: Partners
 ---
 
-This documentation describes the Datadog Nozzle for Pivotal Cloud Foundry (PCF). The Datadog Nozzle
-enables PCF users and administrators to monitor the health and performance of their PCF clusters.
+This documentation describes Datadog Cluster Monitoring for Pivotal Cloud Foundry (PCF). The Datadog Cluster Monitoring combines our Datadog Nozzle with the Datadog Agent, and enables PCF users and administrators to monitor the health and performance of their PCF clusters.
 
-For comprehensive monitoring, you can also install the [Datadog Agent](https://docs.datadoghq.com/integrations/cloud_foundry/#install-the-datadog-agent-bosh-release) and [BOSH Health Monitor](https://docs.datadoghq.com/integrations/cloud_foundry/#configure-the-datadog-plugin-for-bosh-health-monitor).
+For comprehensive monitoring, you can also install the [BOSH Health Monitor](https://docs.datadoghq.com/integrations/cloud_foundry/#configure-the-datadog-plugin-for-bosh-health-monitor).
 
 ## <a id='overview'></a>Overview
 
-Datadog Nozzle for PCF is a Cloud Foundry component which forwards metrics from the Loggregator Firehose to the Datadog monitoring platform. Any Cloud Foundry deployment can send metrics and events to Datadog. The data helps you track the health and availability of all nodes in the deployment, monitor the jobs they run, collect metrics from the Loggregator Firehose, and more.
+Datadog Cluster Monitoring for PCF is made up of two components:
+* The Datadog Firehose Nozzle
+* The Datadog Agent
 
-Datadog Nozzle for PCF includes the following key features:
+The Datadog Nozzle is a Cloud Foundry component which forwards metrics from the Loggregator Firehose to the Datadog monitoring platform. Any Cloud Foundry deployment can send metrics and events to Datadog. The data helps you track the health and availability of all nodes in the deployment, monitor the jobs they run, collect metrics from the Loggregator Firehose, and more.
+
+The Datadog Agent is small daemon that is installed via BOSH addon to run on every host and collect infrastructure metrics from each host.
+
+Datadog Cluster Monitoring for PCF includes the following key features:
```

Comment:
Can we link from here to the build pack tile?

Reply:
We can't link to the tile until it's released but when it is let's update the docs to reflect that